src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -33,10 +34,37 @@ class AssetsInstallCommand extends ContainerAwareCommand
     const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
+    private $filesystem;
+
     /**
-     * @var Filesystem
+     * @param Filesystem|null $filesystem
      */
-    private $filesystem;
+    public function __construct($filesystem = null)
+    {
+        parent::__construct();
+
+        if (!$filesystem instanceof Filesystem) {
```

Comment:
actually it is.. https://github.com/symfony/symfony/blob/v3.3.6/src/Symfony/Component/Console/Command/Command.php#L51